src/server/routes/register.ts

```diff
@@ -140,7 +139,7 @@ const OktaRegistration = async (
 		return res.sendStatus(418);
 	}
 
-	const { email = '', _cmpConsentedState = false } = req.body;
```

Comment:
if you're removing the `_cmpConsentedState` here, is it also possible to remove it from the form body if its not being used? think there is a component that adds it in